src/ConnectorRuntime.ts

```diff
@@ -223,9 +223,10 @@ export class ConnectorRuntime extends AbstractConnectorRuntime<ConnectorRuntimeC
         }
 
         const httpServer = config.infrastructure.httpServer as any;
-        if (httpServer?.apiKey) httpServer.apiKey = "redacted, but enabled";
-        if (httpServer?.oidc) httpServer.oidc = "redacted, but enabled";
-        if (httpServer?.jwtBearer) httpServer.jwtBearer = "redacted, but enabled";
+        const authentication = httpServer?.authentication;
+        if (authentication?.apiKeys) authentication.apiKeys = "redacted, but enabled";
```

Comment:
It might make sense to render out the configs with metadata (e.g. validFrom / validTo / id) but of course not the key itself?

Reply:
IMO way to hard to do that for noting.